.idea/codeStyles/Project.xml

```diff
@@ -17,4 +17,4 @@
       <option name="MULTILINE_STRING_CLOSING_QUOTES_ON_NEW_LINE" value="true" />
     </ScalaCodeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Please revert this change

Reply:
I have no idea how this happened, or what it does, to me it just looks like I deleted the line and put it back in, what should I do with it?

Reply:
Its likely git doing something with line breaks, CRLF -> LF. I'll try to fix that here and commit it.